mirrord/config/src/feature/network/incoming.rs

```diff
@@ -312,7 +312,7 @@ pub struct IncomingConfig {
     /// Ports to ignore when mirroring/stealing traffic, these ports will remain local.
     ///
     /// Can be especially useful when
-    /// [`feature.network.incoming.mode`](#feature-network-incoming-mode) is set to `"stealer"
+    /// [`feature.network.incoming.mode`](#feature-network-incoming-mode) is set to `"steal"
     /// `, and you want to avoid redirecting traffic from some ports (for example, traffic from
```

Comment:
```suggestion
    /// [`feature.network.incoming.mode`](#feature-network-incoming-mode) is set to `"steal"`,
    /// and you want to avoid redirecting traffic from some ports (for example, traffic from
```

The line break before the ` causes an extra space inside the code-formatted text:
![image](https://github.com/metalbear-co/mirrord/assets/12370607/fe3d7cfd-af6c-4464-9992-3c730acaa066)